src/plugins/recursion/README.md

```diff
@@ -81,10 +81,186 @@ kdb set /recursive/define/#2/ref/#1 /recursive/define/#5
 Arbitrary additional data can be saved too such as:
 `kdb set /recursive/define/#2/permission admin`
 
+## A more real example
+
+In order to better show how this plugin can be used, a more sophisticated example will be given.
```

Comment:
Its better if the examples show different aspects, not one "less real" and another "more real".